db-utils/src/main/kotlin/com/icerockdev/common/database/sql/Table.kt

```diff
@@ -54,8 +58,8 @@ fun <T : Any> Table.jsonb(name: String, typeRef: TypeReference<T>, jsonMapper: O
  * @param clazz Class object
  * @param jsonMapper Jackson object mapper
  */
-fun <T : Any> Table.jsonb(name: String, clazz: Class<T>, jsonMapper: ObjectMapper): Column<T>
-        = registerColumn(name, Json(clazz, jsonMapper))
+inline fun <reified T : Any> Table.jsonb(name: String, clazz: Class<T>, jsonMapper: ObjectMapper): Column<T> =
```

Comment:
if you use `reified` you can get `class` just from `T` - `T::class.java`. you use inline so use abilities of inline :)